test/tests/ruby-gems/container.sh

```diff
@@ -1,20 +1,20 @@
-#!/bin/bash
+#!/bin/sh
 set -e
 
 # list taken from https://rubygems.org/stats
-gems=(
-	thor
-	rake
-	rails
-	rack
-	activesupport
-	activerecord
-	actionpack
-	json
-	actionmailer
-	activeresource
-)
+set -- \
+	thor \
+	rake \
+	rails \
+	rack \
+	activesupport \
+	activerecord \
+	actionpack \
+	json \
+	actionmailer \
+	activeresource \
 
-for gem in "${gems[@]}"; do
+
+for gem; do
```

Comment:
Looks like this was originally over-complicated; maybe simplify by just putting the list directly into the for loop?